Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -47,7 +47,7 @@ static std::unique_ptr<SoundStream> CreateSoundStreamForBackend(std::string_view
 
 void InitSoundStream(Core::System& system)
 {
-  std::string backend = Config::Get(Config::MAIN_AUDIO_BACKEND);
+  std::string backend = Get(Config::MAIN_AUDIO_BACKEND);
```

Comment:
Yeah, the namespace qualifier here is technically superfluous because it can be inferred from the type of the argument, but the code you're replacing was written by humans to be read by humans. I think the original `Config::Get` is more readable than just `Get`. If you personally find some superfluous qualifiers that you think are hurting readability then it's probably fine to remove those, but I don't think we want big automated replacements like this with no significant benefit.

Reply:
It's not just `Config::Get`, it's `Config::Get(Config::XXX)`. I think `Get(Config::XXX)` makes in clear enough we are getting a config

Reply:
@tygyh I cannot agree. Just because [ADL](https://en.cppreference.com/w/cpp/language/adl) is possible here doesn't mean it should be used. It's pretty unintuitive for anything other than operator overloads.

Reply:
> It already bewildered me back in the [first](https://github.com/dolphin-emu/dolphin/pull/12987) of your automated refactorings.

I remember discussing it with you, but since I couldn't find any PR with a similar name to this one I assumed I just dreamed it. I'm still not convinced, but I'm not dying on this hill